RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -354,7 +354,12 @@ void finalizeMaximize(boolean speculative)
 		addBonusToMaximize($item[Cursed Magnifying Glass], 1000);
 	}
 	addBonusToMaximize($item[mafia thumb ring], 200); // adventures
+	if(get_property("auto_MLSafetyLimit").to_int() >= 25)
+	{
+		addBonusToMaximize($item[carnivorous potted plant], 200); // free kills
+	}
```

Comment:
This is a bad idea. if we ask for ML we will use ML. if a user manually pulled it, it was specifically to use in a ML Desiring zone, not to use everywhere

auto_MLSafetyLimit is not "target ML user always wants". it is "never ever exceed this ML because you keep on dying". it is a workaround for our bad ML code

Reply:
```suggestion
```